launcher/ui/MainWindow.h

```diff
@@ -192,6 +192,8 @@ private slots:
     void keyReleaseEvent(QKeyEvent *event) override;
 #endif
 
+    void on_InstanceState_changed(bool running);
```

Comment:
it's best to avoid naming slots that aren't bound to any UI widget in this scheme, as Qt will try to auto-connect this slow to a UI element with the id `InstanceState` on the `changed` event